pkg/cumulusurllog/cumulusurllog.go

```diff
@@ -0,0 +1,94 @@
+package cumulusurllog
```

Comment:
Is this package new? I would suggest to call it simply `cumulus`

Reply:
cumulus is a sap specific tool, should avoid to add package name or anything related to cuimulus this in open source piper

Reply:
We've already had some discussion about that. Do you have any vision what it can be used further except url log creation?

Reply:
lets use the existing package `log`  and create a new file `url.go`